source/includes/gcp/_target_proxies.md

```diff
@@ -42,11 +42,12 @@ Retrieve a list of all target proxies in an [environment](#administration-enviro
 | -------------------------------- | ------ |
 | `creationTimestamp`<br/>*string* | Creation timestamp in RFC3339 text format. |
 | `id`<br/>*UUID*                  | The id of the backend service. |
-| `kind`<br/>*string*              | Type of the resource. Either compute#targetHttpProxy, compute#targetHttpsProxy, compute#targetSslProxy or compute#TargetTcpProxy for target proxies.|
+| `kind`<br/>*string*              | Type of the resource. Either compute#targetHttpProxy, compute#targetHttpsProxy, compute#targetSslProxy or compute#TargetTcpProxy for target proxies. |
```

Comment:
Remove the SSL and TCP proxy mention, we don't support them (we can list them, but we can't create them)

Reply:
This is under the GET/LIST. Shouldn't we include them here?

Reply:
See below comment.

Reply:
ae81aa9d03315427864b7953c7932bce5c8f5420